### PR TITLE
documentation/ZENKO-3083_xdm_gui_document_adding_a_transient_source_l…

### PR DESCRIPTION
**Pull request by wabernat**
…ocation

Added instructions for adding a transient source location. Did some renaming. Updated links as needed.

### CODE REVIEW

File: docs/docsource/operation/XDM_GUI/location_management/add_a_transient_source_storage_location.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+   Location** to raise the **ADD NEW STORAGE LOCATION** window.
+
+   .. image:: ../../Graphics/xdm_ui_add_new_storage_location.png
+
+#. The **Add New Storage Location** modal appears. Enter the **Location Name**
+   (AWS S3 naming rules apply: lowercase alphanumerics and hyphens only) and
+   select **Scality RING with Sproxyd Connector** from the **Location Type**
+   drop-down list,
+
+   .. image:: ../../Graphics/xdm_ui_add_storage_location_sproxyd.png
```
Comment (lucieleonard) [inline docs/docsource/operation/XDM_GUI/location_management/add_a_transient_source_storage_location.rst:29]: apparently this image breaks the build

**Review by lucieleonard — APPROVED** [review]

File: docs/docsource/operation/XDM_GUI/location_management/add_a_transient_source_storage_location.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/operation/XDM_GUI/location_management/add_a_transient_source_storage_location.rst:29]: yeah...but WHY?

Comment (wabernat) [reply docs/docsource/operation/XDM_GUI/location_management/add_a_transient_source_storage_location.rst:29]: oh... heh... I hadn't committed and pushed the new screengrabs.

**Review by ilkescality — APPROVED** [review]

**Review by miniscruff — APPROVED** [review]

**Pull request merged**